### DISCUSSION TITLE
Search usages

### DISCUSSION THREAD

**Discussion by fmartingr** [go-shiori/shiori#375] [Feature requests]
I would like to hear from the community how do you use or wish to use the search bar in shiori. I recently turned down contribution #285  because an unclear path on how to move forward (among other reasons) so instead of that I would like everyone to discuss how it may work here so we can create some achievable goals.

**Comment by 8bitgentleman** [comment]
For me there are a few different ways I'd like to use/currently use search: 

1. by default right now I filter out all items with a 'read' tag. Not a good user experience but it gets the job done
2. To search for tags there could be a specific check box that limits the search to only tags or go the route that pocket does and have a completely separate menu
3. Full text search. This is a big one and I'd imagine difficult to implement efficiently. More often than not I remember something I read but not explicitly what article it comes from. Full text search would allow me to search across all of my saved articles for one that contains a specific word or phrase
